ec2/development/tthyer/sc-ec2-linux-jumpcloud-development-tthyer.yaml

```diff
@@ -484,7 +437,7 @@ Resources:
             /usr/bin/apt-get install -y python-pip
             /usr/bin/apt-get install -y python-setuptools
             /bin/mkdir -p /opt/aws/bin /opt/sage/bin
-            /usr/bin/python /usr/lib/python2.7/dist-packages/easy_install.py --script-dir /opt/aws/bin https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz
+            /usr/bin/python2.7 /usr/lib/python2.7/dist-packages/easy_install.py --script-dir /opt/aws/bin https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz
```

Comment:
this also wasn't updated in previous PR (because this template had changes in flight on this branch)

Reply:
should be able to remove once we do https://sagebionetworks.jira.com/browse/SC-78